specification/synapse/resource-manager/Microsoft.Synapse/stable/2021-06-01/bigDataPool.json

```diff
@@ -554,12 +555,20 @@
         "minExecutors": {
           "type": "integer",
           "format": "int32",
-          "description": "The minimum number of executors alloted"
+          "description": "The minimum number of executors alloted",
+          "x-ms-mutability": [
+            "create",
+            "update"
+          ]
```

Comment:
Can you explain the intent of this change? My reading is that it allows the property to be set by the user either when creating or updating the resource, but does not allow reading -- i.e. that the property is not returned in a read request. This is highly unusual for a property that is not a "secret". Is this really the intent?

Even if this is accurate, I wonder what result you expect to achieve. I know this will have no effect on the generated REST API documentation and I do not think it will result in any changes to the generated SDKs. Can you explain?